changelog/8.7.1.yaml

```
@@ -0,0 +1,88 @@
+version: 8.7.1
+entries:
+    - kind: bug-fix
+      summary: Fix parsing of paths from container-paths.yml file
+      description: |
+        Fix parsing of paths from the `container-paths.yml` file used internally
+        when `elastic-agent container` is run.
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2340
+      issue: []
+      timestamp: 1677780422
+      file:
+        name: 1677780422-fix-container-paths-config-parsing.yaml
+        checksum: 99d209f37ce7d6c57d4d1d3f67016f22696faf28
+    - kind: feature
+      summary: feature-flags
+      description: |
+        Add support for feature flags, starting with one to toggle fully qualified
+        domain name (FQDN) reporting in events generated by Elastic Agent components,
+        via the `host.name` field.
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2218
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2185
+      timestamp: 1679422558
+      file:
+        name: 1679422558-feature-flags.yaml
+        checksum: f6a2c45b76df4b708f19d25ff569ab0394403eab
+    - kind: bug-fix
+      summary: Fix actions acks delay
+      description: |
+        Fixes a bug that caused delayed "ack" response following Osquerybeat query
```

```suggestion
        Fix for OSQuery live query results taking up to five minutes to show up in Kibana.
```

This matches the language in the known issue for this bug https://www.elastic.co/guide/en/fleet/current/release-notes-8.7.0.html

---

I think we should make the description text here the same as your suggestion in https://github.com/elastic/ingest-docs/pull/150#discussion_r1172004810?

---

> I think we should make the description text here the same as your suggestion in [elastic/ingest-docs#150 (comment)](https://github.com/elastic/ingest-docs/pull/150#discussion_r1172004810)?


Done in 8f76938cfa88714c08bce9e5357f9ba6ba09c606.